.github/workflows/ci.yml

```diff
@@ -49,10 +49,7 @@ jobs:
 
     if: github.repository_owner == 'files-community'
 
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: pwsh
+    runs-on: windows-latest
```

Comment:
What's the change for?

Reply:
Just out of interest 😅

Reply:
Just reckoned why it should be Ubuntu while the default shell is specified to be pwsh.

Reply:
Isn't ubuntu far cheaper?

Reply:
In what aspect? Set up finishes in a matter of seconds.

Reply:
It costs less credits to use Linux, and considering that Files uses Windows for most of its actions I'm sure it's not got loads left.

Reply:
Anyways, doesn't really matter.

Reply:
Whichever is quicker and uses the least resources is preferred.

Reply:
As per our conversation on Discord, let's switch back to Linux.

Reply:
@heftymouse made a good point, I'd forgotten the credit limit only applies to private repos. Is windows that much quicker?

Reply:
I haven't compared but the setup finishes 1 sec.